tests/pagination-display.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import { PaginationDisplay } from '../src/ts';
+
+describe('Banner', () => {
+  it('should match snapshot', () => {
```

Comment:
More descriptive name plz.

Reply:
Also, a few more snapshots for this would be nice.

Reply:
Updated the description name.

Reply:
It does not have much to display as it is a very simple component. Instead, I will add more snapshots to the `Pagination` component.

Reply:
Actually `Pagination` component has many snapshots.